releasenotes/notes/tcp-queue-length-max-fill-rate-492a5b1a6444a609.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every releasenote are combined when the
```

Comment:
We should add something about deprecating `only_count_nb_contexts`. Also have we look into the customer impact in changing those metrics ? We don't want to trigger monitor or break dashboard when someone upgrade.

Reply:
I’ve added a mention that the `only_count_nb_contexts` parameter has been removed.
And I changed the release note section to `upgrade`.
Is that OK ?

Reply:
:+1: